compiler/damlc/tests/daml-test-files/ContractKeyNotEffective.daml

```diff
@@ -1,6 +1,7 @@
--- @ERROR range=14:1-14:19; contract 0053db8d12a2aebf02e9d3c2dc653c7b8f61914eeef91b2d6e08b5b066925592a4 not effective, but we found its key!
+-- @ERROR range=26:1-26:19; contract 000fffe8781952ace1141fabd7d28d48c8a157788c53392a6e698a534f89ee2779 not effective, but we found its key!
```

Comment:
I think it's better if we just check for substring `not effective, but we found its key!` while leaving a comment

Reply:
Yeah, I wasn't sure if the contract id had any significance, I'm also not entirely sure why it changed - could it be the script dependence? Or is the contract id dependent on the entire contents of the file, rather than just the "onchain" bit